array-manipulation-3/src/flatten.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-unused-vars */
+function flatten(array) {
+  var flat = [];
+  for (var i = 0; i < array.length; i++) {
+    if (typeof array[i] !== 'object') {
+      flat.push(array[i]);
```

Comment:
`Array.isArray()` would be more reliable.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray